ubuntu/debian/control

```diff
@@ -68,6 +68,7 @@ Architecture: any
 Section: libs
 Pre-Depends: ${misc:Pre-Depends}
 Depends: ${shlibs:Depends}, ${misc:Depends}
+Recommends: python3-gz-sim8
```

Comment:
`libgz-sim8-plugins'  is going to depend on `libgz-sim8` via `shlibs` so we can save this one. I do no find it on sim7.

Reply:
We've added the ability to write Gazebo systems (plugins) in python in gz-sim8 (see https://github.com/gazebosim/gz-sim/blob/gz-sim8/src/systems/python_system_loader/PythonSystemLoader.cc), and users who write python systems will need the python bindings in `python3-gz-sim8`.